### PR TITLE
WID-688 Feat: Actions Debugger

### PR DESCRIPTION
**Pull request by andy-t-wang**
Rebased actions debugger. 
- Use JSON5 to allow more friendly input formats
- Reset state to idle between calls so there's visual indicator of something running.
- 
Screenshot 2024-01-30 at 12 42 31 PM

### CODE REVIEW

File: web-app-router/scenes/Portal/Teams/TeamId/Apps/AppId/Actions/ActionId/ProofDebuging/Debugger/index.tsx [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+      <div className="w-full grid gap-y-6">
+        <h1 className="text-lg font-[550]">Output parameters</h1>
+        <div className="grid gap-y-8">
+          <Input
+            register={register("signal")}
+            helperText="Enter the signal as passed to IDKit"
+            label="Signal"
+            placeholder="my_signal"
+          />
+          <TextArea
```
Comment (igorosip0v) [inline web-app-router/scenes/Portal/Teams/TeamId/Apps/AppId/Actions/ActionId/ProofDebuging/Debugger/index.tsx:187]: @andy-t-wang What do you think about to try something like [embedded editor](https://www.npmjs.com/package/@monaco-editor/react) here

File: web-app-router/scenes/Portal/Teams/TeamId/Apps/AppId/Actions/ActionId/ProofDebuging/Debugger/index.tsx [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+          type: "manual",
+          message: "Unable to auto format the response",
+        });
+      }
+    },
+    [setValue, setError]
+  );
+  return (
+    <form
+      className="w-full grid-cols-2 grid items-start justify-between gap-x-32"
```
Comment (igorosip0v) [inline web-app-router/scenes/Portal/Teams/TeamId/Apps/AppId/Actions/ActionId/ProofDebuging/Debugger/index.tsx:175]: @andy-t-wang Same as [here](https://github.com/worldcoin/developer-portal/pull/511#discussion_r1472796613)

File: web-app-router/scenes/Portal/Teams/TeamId/Apps/AppId/Actions/ActionId/ProofDebuging/Debugger/index.tsx [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+                  ? "success"
+                  : "error"
+            }
+          >
+            {status === Status.IDLE && <WorldcoinIcon />}
+            {status === Status.SUCCESS && (
+              <CheckIcon className="text-system-success-500" />
+            )}
+            {status === Status.ERROR && (
+              <CloseIcon className="text-system-error-500 stroke-3 w-5 h-5" />
```
Comment (igorosip0v) [inline web-app-router/scenes/Portal/Teams/TeamId/Apps/AppId/Actions/ActionId/ProofDebuging/Debugger/index.tsx:229]: @andy-t-wang Coloring of the containing icon is included in `CircleIconContainer`. Let's remove colors from here, it somehow breaks coloring, and the icon is black instead.

Also stroke-3 classname not working at all as I can see.

**Review by igorosip0v — APPROVED** [review]

**Comment by andy-t-wang** [comment]
Add some margin to the text input

File: web-app-router/scenes/Portal/Teams/TeamId/Apps/AppId/Actions/ActionId/ProofDebuging/Debugger/index.tsx (hunk 1 above)
Comment (andy-t-wang) [reply web-app-router/scenes/Portal/Teams/TeamId/Apps/AppId/Actions/ActionId/ProofDebuging/Debugger/index.tsx:187]: I tried it and found it quite a pain to style and match the existing spec. Perhaps something we revisit in the future. For now the format code button should help.

**Pull request merged**